contentcuration/contentcuration/frontend/channelEdit/views/ResourcePanel.vue

```diff
@@ -196,6 +244,16 @@
       isResource() {
         return !this.isTopic && !this.isExercise;
       },
+      isImported() {
+        return this.node.original_channel_id !== this.channelId;
+      },
+      importedChannelLink() {
+        // TODO: Eventually, update with this.node.original_source_node_id for correct path
+        let channelId = this.node.original_channel_id;
+        let parentId = this.node.original_parent_id;
+        let originalNodeId = this.node.original_node_id;
+        return `/channels/${channelId}/#/${parentId}/${originalNodeId}`;
```

Comment:
this would be made more robust by going through the [Vue router](https://router.vuejs.org/api/#router-resolve). For example:

```js
const clientPath = router.resolve(
{
  name: RouterNames.TREE_VIEW,
  params: {
    nodeId: parentId,
    detailNodeId: originalNodeId,
  }
)
return `/channels/${channelId}/#/${clientPath}`;
```